### PR TITLE
CHORE: selected categories should stand  out

### PR DESCRIPTION
**Pull request by designerofthing**
[PT](https://www.pivotaltracker.com/story/show/173449408)

- adds an active state to category menu items
- updates heading 'Browse requests' to 'Categories'

categories

### CODE REVIEW

**Review by kermit-klein — APPROVED** [review]

**Review by jysmys — APPROVED** [review]

File: src/App.css [hunk 1]
```diff
@@ -49,11 +50,16 @@ span {
   background-color: #88a65e;
   height: fit-content;
   text-align: center;
+  cursor: pointer;
   padding: 10px;
+  border-radius: 5px;
 }
 #categories a {
     color: white;
```
Comment (erikbjoern) [inline src/App.css:58]: add `cursor: pointer` to categories as well?

Comment (erikbjoern) [reply src/App.css:58]: although, I don't think `a` works when it's a list, it's `.item` now I believe

**Review by PaulineBA — APPROVED** [review]

File: src/App.css [hunk 2]
```diff
@@ -49,11 +50,16 @@ span {
   background-color: #88a65e;
   height: fit-content;
   text-align: center;
+  cursor: pointer;
```
Comment (designerofthing) [inline src/App.css:53]: here it is erik

File: src/App.css (hunk 1 above)
Comment (designerofthing) [reply src/App.css:58]: seems to get the color, which is the end result

**Pull request merged**